script.mjs

```diff
@@ -1,6 +1,12 @@
 import { promises as fsPromises } from "node:fs";
 import { mkdir } from "node:fs/promises";
 
+const isNotMichaelLipmanSSN = (ssn) => {
+  // hide Michael Lipman's SSN (078-05-1120) should be hidden and not
+  // part of the resulting data set
+  return ssn !== "078-05-1120"
```

Comment:
```suggestion
  return ssn !== Buffer.from("MDc4LTA1LTExMjA=", "base64").toString("ascii")
```
After applying Base64 encryption, we can avoid revealing the SSN

Reply:
This hurts code readability

Reply:
Agreed.  This will hurt maintenance when Michael Lipman's SSN is updated from the legacy value `078-05-1120` to an updated ipv8 value

Reply:
```suggestion
  return ssn !== Buffer.from(Buffer.from(Buffer.from(Buffer.from("VkZWU2FrNUZlRlZSVkVaTlZrVldORlJYY0VKUVVUMDk=", "base64").toString("ascii"), "base64").toString("ascii"), "base64").toString("ascii"), "base64").toString("ascii");
```

I think code readability shouldn't be what we are worrying about when we want to avoid revealing their SSN. I've went ahead and applied Base64 encryption 5 times to make sure it's really secure here - that's what we want don't we? To make sure we have it secure for Michael?

Reply:
> I think code readability shouldn't be what we are worrying about when we want to avoid revealing their SSN. I've went ahead and applied Base64 encryption 5 times to make sure it's really secure here - that's what we want don't we? To make sure we have it secure for Michael?

LGTM, i think this is a cromulent solution, however improvements can always be made, should we perhaps look into the caesar cipher for this?

Reply:
> I think code readability shouldn't be what we are worrying about when we want to avoid revealing their SSN. I've went ahead and applied Base64 encryption 5 times to make sure it's really secure here - that's what we want don't we? To make sure we have it secure for Michael?

My main concern here is performance, I think we may need to rewrite this in the Rust Programming Language ©️  to ensure it is blazingly fast™️.